### PR TITLE
Split out embedded integration from connected integration

### PR DESCRIPTION
**Pull request by cwarnermm**
Generated previews are available for the following updated pages:
- [Mattermost for Microsoft Teams](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6766/about/mattermost-for-microsoft-teams.html)
- [Install the Mattermost for Microsoft Teams plugin](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6766/about/install-mattermost-for-microsoft-teams-plugin.html)
- [Collaborate within connected Microsoft Teams](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6766/collaborate/collaborate-within-connected-microsoft-teams.html)

Embedded integration content has been split out from plugin details:
- [Embed Mattermost within Microsoft Teams](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6766/about/embed-mattermost-within-microsoft-teams.html)
- [Collaborate within embedded Microsoft Teams](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6766/collaborate/collaborate-within-embedded-microsoft-teams.html)

### CODE REVIEW

File: source/about/embed-mattermost-within-microsoft-teams.rst [hunk 1]
```diff
@@ -1,20 +1,15 @@
-Install the Mattermost App in Microsoft Teams
-=============================================
-
+Embed Mattermost within Microsoft Teams
```
Comment (johndavidlugtu) [inline source/about/embed-mattermost-within-microsoft-teams.rst:1]: Should this be "Embedded Mattermost in Microsoft Teams"? Deferring to you @cwarnermm

File: source/about/embed-mattermost-within-microsoft-teams.rst [hunk 2]
```diff
@@ -1,20 +1,15 @@
...
+=======================================
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
-.. tip::
-
-  Looking to `install the Mattermost for Microsoft Teams plugin </about/install-mattermost-for-microsoft-teams-plugin.html>`__ instead?
-
-With `Mattermost for Microsoft Teams </about/mattermost-for-microsoft-teams.html>`__, you can embed your Mattermost workspace within your Microsoft Teams instance and take advantage of `embedded app features <#benefits-of-the-embedded-app>`_.
+With the `Mattermost for Microsoft Teams </about/mattermost-for-microsoft-teams.html>`__ integration, you can embed your Mattermost workspace within your Microsoft Teams instance and take advantage of `embedded app features <about/mattermost-for-microsoft-teams.html#benefits-of-embedding-mattermost>`__.
```
Comment (johndavidlugtu) [inline source/about/embed-mattermost-within-microsoft-teams.rst:6]: "Embedded Mattermost for Microsoft Teams"

File: source/about/embed-mattermost-within-microsoft-teams.rst [hunk 3]
```diff
@@ -1,20 +1,15 @@
...
 
 .. important::
 
-    You must `install the Microsoft Teams plugin in Mattermost </plugins/install-mattermost-for-microsoft-teams-plugin>`__ before you can embed the Mattermost app within Microsoft Teams.
+    You must `install the Microsoft Teams plugin in Mattermost </about/install-mattermost-for-microsoft-teams-plugin.html>`__ before you can embed the Mattermost app within Microsoft Teams.
```
Comment (johndavidlugtu) [inline source/about/embed-mattermost-within-microsoft-teams.rst:10]: How can we differentiate this with the "Microsoft Teams Connector plugin"?

File: source/about/install-mattermost-for-microsoft-teams-plugin.rst [hunk 4]
```diff
@@ -6,9 +6,9 @@ Install the Mattermost for Microsoft Teams plugin
 
 .. tip::
 
-  Looking to `install the Mattermost app within Microsoft Teams </about/install-mattermost-app-in-microsoft-teams.html>`__ instead?
+  Looking to `embed Mattermost within Microsoft Teams </about/embed-mattermost-within-microsoft-teams.html>`__ instead?
 
-To install the `Mattermost for Microsoft Teams </plugins/mattermost-for-microsoft-teams>`__ plugin in Mattermost:
+To install the `Mattermost for Microsoft Teams </configure/plugins-configuration-settings.html#ms-teams-sync>`__ plugin in Mattermost:
```
Comment (johndavidlugtu) [inline source/about/install-mattermost-for-microsoft-teams-plugin.rst:11]: "Embedded Mattermost for Microsoft Teams"

File: source/about/mattermost-for-microsoft-teams.rst [hunk 5]
```diff
@@ -4,13 +4,45 @@ Mattermost for Microsoft Teams
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
-Mattermost for Microsoft Teams enables you to collaborate with technical & operations teams seamlessly through the Mattermost app, without leaving Microsoft Teams. This Mattermost integration includes 2 components:
+Mattermost for Microsoft Teams enables you to collaborate with technical and operations teams seamlessly through the Mattermost app, without leaving Microsoft Teams. Two integrations are available: a `connected integration <#connect-mattermost-with-microsoft-teams>`__ and an `embedded integration <#embed-mattermost-within-mattermost-teams>`__.
```
Comment (johndavidlugtu) [inline source/about/mattermost-for-microsoft-teams.rst:7]: "a connected experience" "embedded experience"

In docs, do we differentiate between integration and plugin?

File: source/collaborate/collaborate-within-connected-microsoft-teams.rst [hunk 6]
```diff
@@ -1,15 +1,13 @@
...
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
 .. |plus-icon| image:: ../images/plus_F0415.svg
   :alt: Open menus using the plus icon.
 
-`Mattermost for Microsoft Teams </about/mattermost-for-microsoft-teams.html>`__ enables you to collaborate with technical & operations teams seamlessly through the Mattermost app, without leaving Microsoft Teams.
-
-Join us on `Mattermost Academy <https://academy.mattermost.com/>`__ to enroll in the `Mattermost for Microsoft Teams <https://academy.mattermost.com/p/mattermost-for-microsoft-teams>`__ training course where you'll learn how to start collaborating with colleagues across both systems.
+The `Mattermost for Microsoft Teams </about/mattermost-for-microsoft-teams.html>`__ integration enables you to collaborate with technical & operations teams seamlessly through the Mattermost app, without leaving Microsoft Teams.
```
Comment (johndavidlugtu) [inline source/collaborate/collaborate-within-connected-microsoft-teams.rst:10]: The Embedded Mattermost for Microsoft Teams

File: source/about/embed-mattermost-within-microsoft-teams.rst (hunk 2 above)
Comment (johndavidlugtu) [reply source/about/embed-mattermost-within-microsoft-teams.rst:6]: Should we capitalize "embedded"?

File: source/collaborate/collaborate-within-embedded-microsoft-teams.rst [hunk 7]
```diff
@@ -0,0 +1,16 @@
+Collaborate within embedded Microsoft Teams
+===========================================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+The `Mattermost for Microsoft Teams </about/mattermost-for-microsoft-teams.html>`__ integration enables you to collaborate with technical & operations teams seamlessly through the Mattermost app, without leaving Microsoft Teams.
```
Comment (johndavidlugtu) [inline source/collaborate/collaborate-within-embedded-microsoft-teams.rst:7]: Embedded Mattermost for Microsoft Teams

**Review by johndavidlugtu — CHANGES_REQUESTED** [review]

File: source/about/embed-mattermost-within-microsoft-teams.rst (hunk 1 above)
Comment (cwarnermm) [reply source/about/embed-mattermost-within-microsoft-teams.rst:1]: "Embed" here is intended as a verb to align with the docs style guide. (Headings and titles aim to be imperative and start with a verb for consistency).

File: source/about/mattermost-for-microsoft-teams.rst (hunk 5 above)
Comment (cwarnermm) [reply source/about/mattermost-for-microsoft-teams.rst:7]: Historically, the docs referred to plugins and integrations somewhat interchangeably. Docs aim to refer to plugins as "Mattermost integrations" to reach a wider audience. The term "plugin" is reserved for technical developer documentation.

File: source/about/embed-mattermost-within-microsoft-teams.rst (hunk 2 above)
Comment (cwarnermm) [reply source/about/embed-mattermost-within-microsoft-teams.rst:6]: This page is specific to the embedded experience, and this paragraph is introducing the connection between the integration itself and the embedded feature.

File: source/about/embed-mattermost-within-microsoft-teams.rst (hunk 3 above)
Comment (cwarnermm) [reply source/about/embed-mattermost-within-microsoft-teams.rst:10]: It's my understanding that admins need to install the MS Teams integration in order to be able to embed Mattermost in an MS Teams instance. Is that understanding correct?

**Comment by cwarnermm** [comment]
@johndavidlugtu - I've applied your feedback; however, I think we have a few threads on this PR that may need further discussion/alignment (especially since I'm working from an understanding that the embedded experience requires the plugin itself to be installed, and your feedback suggests to be that this may not be the case).

File: source/about/embed-mattermost-within-microsoft-teams.rst (hunk 3 above)
Comment (johndavidlugtu) [reply source/about/embed-mattermost-within-microsoft-teams.rst]: Correct. See latest note for more info

**Comment by johndavidlugtu** [comment]
@cwarnermm good callout!

Yes, it does use the [same integration](https://github.com/mattermost/mattermost-plugin-msteams-sync). After thinking it through a bit more based on what you raised, I'm not sure there's value in highlighting the fact it's the same integration that needs to be installed for the MS Teams connected experience (syncing) and the MS Teams embedded experience.

**Comment by cwarnermm** [comment]
@johndavidlugtu - The one place where I think we need to be clear that it's the same integration is during install/setup. Admins must install the plugin first to be able to embed Mattermost in MS Teams. Thoughts?

**Comment by johndavidlugtu** [comment]
@cwarnermm yup I think that's a good place to mention. Thanks!

**Review by johndavidlugtu — APPROVED** [review]
Thanks @cwarnermm !

**Pull request merged**